package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ltonetwork/relay",
-  "version": "0.0.0",
```

Comment:
The version is set during `docker build` based on the git tag. In the source code it should remain '0.0.0'.